### ISSUE TITLE
Suppressed Count Not Displayed on Project Overview

### ISSUE THREAD

**Issue by msymons** [DependencyTrack/dependency-track#300]
### Issue Type:
  - [X] defect report
  - [ ] enhancement request
  
  
### Current Behavior:
After performing vulnerability triage and suppressing some vulnerabilities then the "Suppressed" count on the project's "Overview" does not reflect the count of suppressions...  it always displays 0.  However, the total suppressed count does display on the DT Dashboard.

### Steps to Reproduce (if defect):
* Create a project and upload a BOM that includes at least one vulnerability (or just choose one of the existing projects that has vulnerabilities > 0.
* Navigate to project -> Audit
* Select a vulnerability and click to open the triage section
* Perform a standard suppression.  ie, add comment, select analysis = (say) false positive & click "suppress".
* Go and make a cup of coffee (ie, wait for DT to update project analysis)
* Project Overview has not updated the Suppress count.

![image](https://user-images.githubusercontent.com/4938718/54772971-7e0cf480-4c00-11e9-88e5-7d8a93cb0e22.png)

### Expected Behavior:
Suppress Count on project overview should reflect correct count.

### Environment:
  
  - Dependency-Track Version: 3.4.0
  - Distribution: [Executable WAR]
  - BOM Format & Version: CycloneDX v1.0
  - Database Server: [PostgreSQL ]
  - Browser: Firefox

**Comment by stevespringett** [comment]
Project metrics are not calculated in real time. There is a schedule for them and the last time metrics were run is displayed beneath suppressed (last measurement). This can be manually triggered using the refresh icon. 

Are you stating that even after metrics have run that the suppression count is not updated? If so, that would be a defect. But if its calculated correctly, just not instantaneously, then that would not be a defect and is expected behavior.

**Comment by msymons** [comment]
I know that there is a delay  (hence reference to going to make a cup of coffee).  I did manually kick off the metrics recalculation (for more than one project) by using the refresh icon... and it was then that I noticed that the "Suppressed" count was incorrect.

I will double check my "projects with suppressions"  tomorrow and see whether the counts in the project overviews are still all 0.    As I said, the DT dashboard seems to be showing the correct count that sums suppressions across all projects.
